docs/nuxt/layouts/docs.vue

```diff
@@ -5,6 +5,7 @@
         <div class="container-fluid">
             <div class="row flex-xl-nowrap2">
                 <div class="col-12 col-md-3 col-xl-2 bd-sidebar">
+                    <div class="d-block d-sm-none py-4 mt-3"></div>
```

Comment:
Temporary hack to push search & toggler in view on XS screens

Reply:
no positive effect

Reply:
![image](https://user-images.githubusercontent.com/20986566/29835058-f56c6048-8d1a-11e7-8050-7e24f9a5a40e.png)
toggler showed, but its not clickable

Reply:
ah may need to add v-b-toggle directive...

Reply:
My mistake.  I had component `<nav-toggle>` instead of `<b-nav-toggle>`

Reply:
Note search currently doesn't work from what I can see.  Its just there for when search will be applied.

I wonder if that should be hidden, or disabled?